### ISSUE TITLE
Callbacks and doing things with them

### ISSUE THREAD

**Issue by yratof** [dixonandmoe/rellax#239]
Has anyone been able to use the callback to change anything outside of positions? It seems like it's not doing anything other than bringing back X & Y coordinates.

Examples welcomed here

**Comment by sendy34** [comment]
@yratof 
Yes, for that you can bind a specific element with Rellax, so that you know about callback positions for that particular element. The positions are percentage based are generally revolves around center percentage `data-rellax-percentage="0.5"`.

When you know that callback is for a specific element by binding it manually, you def can add some JS code based on positions.

It would be def great if Rellax would add additional elements in callback.
